### DISCUSSION TITLE
How to sign for NFT

### DISCUSSION THREAD

**Discussion by FatherOfLove** [trustwallet/wallet-core#2863] [Q&A]
Is trust wallet core library support for signing the NFT? If yes and how to do that

**Comment by weixuefeng** [comment]
you can build ERC721Transfer, ERC1155Transfer
<img width="525" alt="image" src="https://github.com/trustwallet/wallet-core/assets/25627103/20362fe6-1ff6-4183-8397-25f10a6c7619">
